src/commands/karma.ts

```diff
@@ -0,0 +1,27 @@
+import { Command } from 'discord-akairo';
+import { Message } from 'discord.js';
+import Repository from '../repository/Repository';
+import { UserKarmaInfo } from '../types';
+import { getEmbedFromUserKarma } from '../util';
+
+export default class LeaderboardCommand extends Command {
+    constructor() {
+        super('karma', {
+           aliases: ['karma']
+        });
+    }
+
+    async exec(message: Message) {
+        const { author } = message;
+        const userKarma: UserKarmaInfo[] | undefined = await Repository.getUserKarmaInfo(author.id);
+        if (!userKarma) {
+            return message.reply(`You don't have any karma points!! start helping to get karma!`);
```

Comment:
Can we start by separating string messages like these, away from code? Would be easier changing these later, if we need to update our communication / messaging. This string should be read from an external file (can be JSON / YAML).

Reply:
Will add `const` file for these strings.